electron-app/src/components/inputs/AssetSelect.vue

```diff
@@ -60,6 +70,19 @@ export default class AssetSelect extends Vue {
   @Emit()
   input(_value: string) {}
 
+  get assets(): SupportedAsset[] {
+    if (this.items) {
```

Comment:
Now that I see it I really like the idea more :+1:  thanks for suggesting it :smile: